frontend/packages/ui/src/styles/variables.css

```diff
@@ -13,10 +13,10 @@
   --z-index-dropdown-menu: 1100;
   --z-index-popover: 1500;
   --z-index-tooltip: 2000;
-  --z-index-toast: 3000;
   --z-index-overlay: 3100;
   --z-index-toolbar-opened: 3200;
   --z-index-command-palette: 3300;
+  --z-index-toast: 3400;
```

Comment:
The Toast z-index needs to be greater then CommandPalette one, otherwise the Toast will be hidden by the dialog overlay, like the following image.

<img width="1095" height="964" alt="Screenshot 0007-08-08 at 9 06 50" src="https://github.com/user-attachments/assets/afdcdc6c-18d7-46a0-ac83-b7d8e6fa8d62" />